examples/kubernetes/proxy_with_sa_key.yaml

```
@@ -57,6 +57,20 @@ spec:
           mountPath: /secrets/
           readOnly: true
           # [END cloud_sql_proxy_k8s_volume_mount]
+        # Resource configuration depends on an application's requirements. The
+        # number of connections is correlated to memory use. The amount of IO
+        # across each connection is correlated to CPU use. Users should adjust
+        # the following values based on what their application needs.
+        # For details, see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
```

1. Let's avoid stating the same thing twice
2. "Users" -> "You" to match the rest of the comments

```suggestion
        # Resource configuration depends on an application's requirements.  You
        # should adjust the following values based on what their application needs.
        # For details, see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
```

---

Fixed